core/new-gui/src/app/workspace/service/drag-drop/drag-drop.service.ts

```diff
@@ -228,6 +260,77 @@ export class DragDropService {
         y: ui.offset.top
       }
     });
+
+    if (this.suggestionOperator !== undefined) {
+      const operatorID = this.suggestionOperator.operatorID;
+      const status = true;
+      this.operatorSuggestionUnhighlightStream.next({status, operatorID});
+      this.suggestionOperator = undefined;
+    }
+  }
+
+  private handleOperatorRecommendationOnDrag(): void {
+    let isOperatorDropped = false;
+
+    Observable.fromEvent<MouseEvent>(window, 'mouseup').first()
+      .subscribe(
+        () => isOperatorDropped = true,
+        (error) => console.error(error)
+      );
+
+    Observable.fromEvent<MouseEvent>(window, 'mousemove').auditTime(100).map(
+      value => [value.clientX, value.clientY]
+    ).filter(() => isOperatorDropped === false).subscribe(
+      (mouseCoordinates) => {
+          this.mouseAt = {x: mouseCoordinates[0], y: mouseCoordinates[1]};
+          this.findClosestOperator(this.mouseAt);
+      },
+      (error) => console.error(error)
+    );
```

Comment:
Add comments to this method. In addition, use a local const variable `currentMouseCoordinate` instead of `this.mouseAt`.

`isOperatorDropped === false` can be shortened to `!isOperatorDropped`

Reply:
Fixed it